Packs/CheckPoint_FW/Integrations/CheckPoint_FW/CheckPoint_description.md

```diff
@@ -0,0 +1,39 @@
+Product Name: CheckPoint Firewall Product Type: Network Security Product Version: R80.30
+
+Integration Overview Manage CheckPoint Firewall. Read information and to send commands to the Check Point Firewall server. This integration was integrated and tested with version R80.30 of CheckPoint SmartConsole.
```

Comment:
It was tested on 80.30 but do previous API versions even support these commands?
Is this integration even valid for older versions?

Reply:
This integration will work for all R80.XX versions.